sdk/dapp-kit/src/hooks/wallet/useConnectWallet.ts

```diff
@@ -55,7 +55,12 @@ export function useConnectWallet({
 				);
 				const selectedAccount = getSelectedAccount(connectedSuiAccounts, accountAddress);
 
-				setWalletConnected(wallet, connectedSuiAccounts, selectedAccount);
+				setWalletConnected(
+					wallet,
+					connectedSuiAccounts,
+					selectedAccount,
+					connectResult.supportedIntents,
```

Comment:
If this doesn't exist this is just undefined which is why this is safe, I'm assuming?

Reply:
yup, it's defined as `supportedIntents?: string[]` and is optional everywhere